autom8s/helm-controller.js

```diff
@@ -25,6 +27,9 @@ router.post('/install',
       });
   });
 
+/**
+ * Deletes an already installed chart, identified by its release name
+ */
```

Comment:
Here and everywhere else: when writing a service and returning an HTTP error, don't expose the internal error and avoid returning it. It is not a good practice.
 You should log the error, and if you have a user understandable error message, you may return it.